### PR TITLE
Create entries for request control filter modes

### PR DESCRIPTION
**Pull request by 4mmonium**
This PR adds documentation for the following console variables:

- sv_filterRequestControl
- sv_filterRequestControlSettleTimer

### CODE REVIEW

File: content/docs/server-manual/server-commands.md [hunk 1]
```diff
@@ -229,6 +229,28 @@ A console variable as an integer from 1-5 (default 1); from least to most likely
 
 A console variable as an integer from 1-5 (default 5); from least to most trustworthy (5 being a method such as external three-way authentication).
 
+### `sv_filterRequestControl [mode]`
+
+A console variable used to block `REQUEST_CONTROL_EVENT` routing based on a configurable policy.<br>
+Supported modes for this variable are as follows:
+
+- -1: Default, equivalent to 2 at this time, but will also warn in console so users can fix stuff/be told to disable it.
```
Comment (blattersturm) [inline content/docs/server-manual/server-commands.md:237]: 'so users can fix stuff/be told to disable it' -> don't copy that raw line :-)

File: content/docs/server-manual/server-commands.md [hunk 2]
```diff
@@ -229,6 +229,28 @@ A console variable as an integer from 1-5 (default 1); from least to most likely
...
+- 0: Off. Also disables the routing bucket/entity lockdown-based policy.
+- 1: Blocks control requests to entities controlled by players (currently, occupied vehicles only) that have existed for more than `sv_filterRequestControlSettleTimer` milliseconds (default `30000`) - hereafter referred to as 'settled'.
+- 2: Blocks control requests to all entities controlled by players.
+- 3: Blocks control requests to all entities controlled by players, and any 'settled' non-player entities.
+- 4: Does not route `REQUEST_CONTROL_EVENT` whatsoever.
+
+In addition, any mode but 'off' will have some additional checks as well:
+
+- Control request events can't be routed across routing buckets. This likely wasn't a thing anyway, but just a double-check.
```
Comment (blattersturm) [inline content/docs/server-manual/server-commands.md:246]: Same for the 'This likely wasn't a thing anyway' bit.

File: content/docs/server-manual/server-commands.md [hunk 3]
```diff
@@ -229,6 +229,28 @@ A console variable as an integer from 1-5 (default 1); from least to most likely
...
+- Control request events will always be blocked if the sender is in 'strict' entity lockdown mode, either by the global mode setting, or their routing bucket being set to such.
+
+### `sv_filterRequestControlSettleTimer [time]`
+
+A console variable that allows you to set after how long (based on entity creation time) an entity should be blocked from a `REQUEST_CONTROL_EVENT`. This will only apply to filter request control modes [1 and 3](#svfilterrequestcontrol-mode), which are detailed under `sv_filterRequestControl` in this page.
```
Comment (blattersturm) [inline content/docs/server-manual/server-commands.md:251]: \+ `in milliseconds`

**Review by blattersturm — CHANGES_REQUESTED** [review]

File: content/docs/server-manual/server-commands.md (hunk 1 above)
Comment (4mmonium) [reply content/docs/server-manual/server-commands.md]: Ah, funny how I missed that even though I proof read it twice 😅
Anyway, left as _-1: Default, equivalent to 2 at this time, but will also warn in console_. I ideally was going to add a 'for debug purposes in there' after 'warn in console', but I didn't know if it fitted that criteria.

File: content/docs/server-manual/server-commands.md (hunk 2 above)
Comment (4mmonium) [reply content/docs/server-manual/server-commands.md]: Yep, removed!

File: content/docs/server-manual/server-commands.md (hunk 3 above)
Comment (4mmonium) [reply content/docs/server-manual/server-commands.md]: Changed to: https://github.com/citizenfx/fivem-docs/pull/318/files#diff-331e99259d1395f290e695c1c52cfb483254b82a1a1534aadcaabc08ad0d8e5dR249

Sorry, squashed the commits and force pushed to have everything as one commit, I probably should have done that after this commit so diffs are easier to read, my bad.

**Review by blattersturm — APPROVED** [review]
OK. Can someone still write a draft for the change notification? I'll probably merge the changes *after* next recommended bump.

**Pull request merged**